external/derby/README.md

```diff
@@ -0,0 +1,14 @@
+## Running External tests locally
+To run any AQA tests locally, you follow the same pattern:
+
+0. Ensure your test machine is set up with [test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md).  For external tests, you do need Docker installed.
+
+1. Download/unpack the SDK that you want to test to your test machine
+1. `export TEST_JDK_HOME=</pathToWhereYouInstalledSDK>` 
+1. `git clone https://github.com/AdoptOpenJDK/openjdk-tests.git` 
+1. `cd openjdk-tests`
+1. `./get.sh`
+1. `cd TKG`
+1. export required environment variables, BUILD_LIST and EXTRA_DOCKER_ARGS (example: `export BUILD_LIST=external/jacoco` and `export EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk"`
+1. `make compile`              (This fetches test material and compiles it, based on build.xml files in the test directories)
+1. `make _derby_test`   (When you defined BUILD_LIST to point to a directory in [openjdk-tests/external](https://github.com/AdoptOpenJDK/openjdk-tests/tree/master/external), then this is a testCaseName from the playlist.xml file within the directory you chose)
```

Comment:
Please add the following as the last section:

When [running these from the command-line](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/userGuide.md#local-testing-via-make-targets-on-the-commandline), these tests are grouped under a make target called 'external', so 'make external' would run the entire set of tests found in the openjdk-tests/external directory. This is unadvisable! Limit what you compile and run, BUILD_LIST=external/`<someSubDirectory>`, and TARGET=`<testCaseNameFromSubdirPlaylist>`
These tests run regularly and results can be found in [TRSS Third Party Application view](https://trss.adoptopenjdk.net/ThirdPartyAppVie).